Sources/Files.swift

```diff
@@ -528,6 +528,43 @@ public final class File: FileSystem.Item, FileSystemIterable {
         
         try write(data: data)
     }
+
+    /**
+     *  Append data to the end of the file
+     *
+     *  - parameter data: The data to append to the file
+     *
+     *  - throws: `File.Error.writeFailed` if the file couldn't be written to
+     */
+    public func append(data: Data) throws {
+        do {
+            let handle = try FileHandle(forWritingTo: URL(fileURLWithPath: path))
+            defer {
+                handle.closeFile()
```

Comment:
You don't need to use defer here since no `throwing` expressions occur after opening the `FileHandle`. Just close it at the end 🙂